demo/docbot/cluster_bootstrap.py

```diff
@@ -204,38 +207,39 @@ def initialize_model(client: MLClient) -> None:
 
 
 
-def initialize_ingestion_pipeline(client: MLClient):
+def initialize_ingestion_pipeline(client: MLClient, pipeline="cohere-ingest-pipeline", pipeline_data=None):
```

Comment:
Here we can remove pipeline data. Again below in the description we can make it more modular by including our variable pipeline name in the description.